make/framework.mk

```diff
@@ -57,8 +57,8 @@ FRAMEWORK_PUBLIC_HEADERS = $(FRAMEWORK_HEADERS)
 endif
 
 FRAMEWORK_DIR = $(DIST_FRAMEWORK_DIR)/$(FRAMEWORK_NAME).xcframework
-FRAMEWORK_HEADER = $(BUILD_DIR)/$(FRAMEWORK_NAME).h
-MODULE_MAP = $(BUILD_DIR)/module.modulemap
+FRAMEWORK_HEADER = $(BUILD_DIR)/$(FRAMEWORK_NAME)/$(FRAMEWORK_NAME).h
+MODULE_MAP = $(BUILD_DIR)/$(FRAMEWORK_NAME)/module.modulemap
```

Comment:
I've changed the `MODULE_MAP` path so that it's different for each framework that is built by this make file. Without this change, the frameworks that are built as part of the `build_subset_frameworks.sh` script all use the `module.modulemap` file that corresponds to the `JRE` framework and hence all have a module name of `JRE`. That is bad because Xcode forbids linking to two modules of the same name.